bin/install-runtime.js

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env node
+
+/* Copyright 2017 Mozilla
```

Comment:
> just curious: is this license comment block required for each file? or is keeping it in LICENSE okay?

This is a complicated question!

In general, a license comment block isn't required in order for a file to be licensed. And in particular, the Apache 2.0 license doesn't explicitly state that you must include such a block, although it does imply it, in [APPENDIX: How to apply the Apache License to your work](http://www.apache.org/licenses/LICENSE-2.0.html#apply), where it states:

> To apply the Apache License to your work, attach the following boilerplate notice… The text should be enclosed in the appropriate comment syntax for the file format.

Nor does the [Mozilla Source Code License Policy](https://www.mozilla.org/en-US/MPL/license-policy/) address this question. The [ASF Source Header and Copyright Notice Policy](http://www.apache.org/legal/src-headers.html) does address it, saying:

> Each source file should include the following license header… License headers allow someone examining the file to know the terms for the work, even when it is distributed without the rest of the distribution. Without a licensing notice, it must be assumed that the author has reserved all rights, including the right to copy, modify, and redistribute.

However, it also states:

> This is strictly an ASF policy. Other projects using the Apache License should still refer to the license's appendix for guidance on applying a header to their source files.

So far my approach has been to include the header in each file, even though it's a pain, both because the license appears to suggest it and because it simplifies reuse of the files in other works.

However, I'm not the expert here, and I bet @gerv has a better sense of the best practice for the org when licensing a work using the Apache 2.0 license.

@gerv: I'm also interested in knowing if Mozilla has a best practice regarding copyright notices, as does the ASF (which requires them to be put into a top-levl NOTICE file rather than into the license headers for individual files).